Ten Elephants/Ten Elephants/RecentsProvider/RecentsProviderProtocol.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  RecentsProviderProtocol.swift
+//  Ten Elephants
+//
+//  Created by Алексей Шерстнёв on 14.12.2021.
+//
+
+import Foundation
+
+protocol RecentsProviderProtocol {
+    typealias mealsCompletion = (Result<Meals, NetworkFetchingError>) -> Void
```

Comment:
Типы с большой буквы
```suggestion
    typealias MealsCompletion = (Result<Meals, NetworkFetchingError>) -> Void
```